provider/datasource_rediscloud_cloud_account.go

```diff
@@ -46,7 +46,7 @@ func dataSourceRedisCloudCloudAccount() *schema.Resource {
 
 func dataSourceRedisCloudCloudAccountRead(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
 	var diags diag.Diagnostics
-	client := meta.(*apiClient)
+	client := meta.(*ApiClient)
```

Comment:
Why does this need to be made public?

Reply:
Just spotted this, can't remember why I did it. Rollback incoming...

Reply:
PR looks much nicer now!